<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LegendEditingWidget</class>
 <widget class="QWidget" name="LegendEditingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Legend font</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="FontWidet" name="legendFontWidget" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="legendVisibilityCheckBox">
     <property name="text">
      <string>Visible legend</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="legendFrameVisiblityCheckBox">
     <property name="text">
      <string>Visible legend frame</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="frameShadowVisibilityCheckBox">
     <property name="text">
      <string>Frame shadow</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Frame corner radius</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="legendFrameCornerRadiusSpinBox"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Legend position</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="LegendLayoutChoiceComboBox" name="legendPosiotionComboBox" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LegendLayoutChoiceComboBox</class>
   <extends>QWidget</extends>
   <header>commonWidgets/legendlayoutchoice/legendlayoutchoicecombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FontWidet</class>
   <extends>QWidget</extends>
   <header>commonWidgets/fontwidet.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
